### ISSUE TITLE
Can you test  testplugin_pi v1.0.225.1 with your OS?

### ISSUE THREAD

**Issue by rgleason** [marcobergman/ais_simulation#12]
Marco (sorry) if you have time, can you test to see if   testplugin_pi v1.0.225.1  works on your OS and report on Cruiser Forum in the thread?  This needs to be done for all OS before I can deploy to other plugins.
See
https://www.cruisersforum.com/forums/f134/opencpn-beta-version-5-7-1-beta-test-1-plugins-271525.html#post3734363

**Comment by marcobergman** [comment]
Yes will work on this and other plugins in the morning. Glad to be able to help.

**Comment by rgleason** [comment]
Marco.  I had it wrong for opencpn.conf

[PlugIns]
CatalogExpert=1

**Comment by marcobergman** [comment]
I have ubuntu-x86_64:**22.10** and for that flavour there are no plugins available. Thought it would be advisable to update to the latest and greatest but apparently not. I'll install another virtual machine and do the tests with that.

**Comment by marcobergman** [comment]
Ok I'm on ubuntu-gtk3-x86_64:20.04 now and I have many plugins that I could test, including testplugin_pi v1.0.225.1. Turning to CF now.
